corehq/apps/hqcase/tests/test_utils.py

```diff
@@ -112,3 +116,20 @@ def get_case(*args, **kwargs):
         yield case
     finally:
         factory.close_case(case.case_id)
+
+
+class TestIsCopiedCase(TestCase):
```

Comment:
Thanks so much for writing tests for even this simple utility. I also appreciate that they were put in this specific module, which makes them easy to find!